arangod/Cluster/ActionBase.cpp

```diff
@@ -171,9 +171,12 @@ void ActionBase::startStats() {
 
 /// @brief show progress on Action, and when that progress occurred
 void ActionBase::incStats() {
-  ++_progress;
+  // OSX clang cannot increment double very rare calls
+  std::mutex m;
```

Comment:
Using a local variable of type std::mutex is nonsense. This protects nothing at all. Use a mutex as a member variable and please use a mutex guard for locking.